### PR TITLE
Updated npm packages

### PR DESCRIPTION
**Pull request by abist**
Updated npm packages to latest versions

### CODE REVIEW

File: build/gulpfile.hygiene.js [hunk 1]
```diff
@@ -64,6 +64,7 @@ const indentationFilter = [
 
 	// except multiple specific files
 	'!**/package.json',
+	'!**/package-lock.json',
```
Comment (anthonydresser) [inline build/gulpfile.hygiene.js]: we don't need a package-lock. we use yarn as our package manager not yarn.

Comment (abist) [reply build/gulpfile.hygiene.js]: i couldnt commit without adding the line, this is for linting

File: package.json [hunk 2]
```diff
@@ -125,23 +125,23 @@
...
-    "istanbul": "^0.3.17",
+    "istanbul": "^0.4.5",
     "jsdom-no-contextify": "^3.1.0",
     "lazy.js": "^0.4.2",
     "merge-options": "^1.0.1",
     "mime": "^1.4.1",
     "minimatch": "^3.0.4",
     "mkdirp": "^0.5.0",
-    "mocha": "^2.2.5",
+    "mocha": "^6.1.4",
```
Comment (anthonydresser) [inline package.json]: You have to be careful when updating this far. It very likely could have breaking changes.

**Comment by anthonydresser** [comment]
A lot of these updates are vscode's dev dependencies that none of us are very familiar with. I would be cautious of upgrading these. I think we should be more targeted with updating "security" issues, but only updating security issues that are actually security issues. Based on look at the output of the compliance check non of them are actually security issues.

**Comment by kburtram** [comment]
Let's discuss a bit more and review each warning in detail before moving forward here.

**Pull request closed without merging**